src/applications/disability-benefits/all-claims/content/disabilityLabels.js

```diff
@@ -0,0 +1,837 @@
+export default {
```

Comment:
Is this list temporary? Will it go away once we've got the back end set up?

Reply:
I think this list will persist even after the backend is set up, but I'm assuming it will need some copy edits.

Reply:
:thinking: I think this could get tricky if we have to maintain two separate sets of data. Especially when we throw in the conceptual matches as well.

I talked with Alastair the other day and we were thinking when we get back a conceptual match, the api would return something like "Post Traumatic Stress Disorder (**PTSD**)" if the query text is "PTSD", the condition name is "Post Traumatic Stress Disorder", and "PTSD" is a conceptual match. Meaning the api would return something like `${conditionName} (${conceptualMatch})`.

How would this work if we ran the result through this list and displayed what was in here?